src/en/03_user_guide.md

```diff
@@ -1,23 +1,156 @@
-# User guide (describe main application features)
+# User Guide
 
-## DB entity tree and context menus
+![Welcome page with labels for the primary menu, utilities menu, walkthroughs](./images/main_pg.png)
 
-## Tabs for different operations (SQL Query, Monitoring, Console etc)
+---
 
-## DB tree context menu items
+## DB Entity Tree and Context Menus
+
+The DB entity tree displays all of the database objects inside a server and is located at the right once a database connection is created.
+
+![tree context menu](./images/tree_context_menus.png)
+
+The **context menus** can be accessed by right clicking on one of the tree nodes. The context menus allow you to make actions to the database. 
```

Comment:
Maybe we should add something like "the context menu has the list of actions available for the selected database tree node". The thing is - this menu have different options available for different tree nodes. DB tables have one set of commands, constraints have another etc.